vantiq-cloud-infra-operations/terraform_azure/new/env-dev/03_main/main.tf

```diff
@@ -129,7 +63,6 @@ module "aks" {
   #  vnet_subnet_id = data.terraform_remote_state.network.outputs.vantiq_aks_node_subnet_kubenet_id
 
   # network profile (optional)
-  docker_bridge_cidr = module.constants.aks_config.docker_bridge_cidr
   #  load_balancer_sku =  module.constants.aks_config.load_balancer_sku
```

Comment:
Will be deprecated.
-  https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster#docker_bridge_cidr